### PR TITLE
Upd 20180326 

### PR DESCRIPTION
**Pull request by novia713**
🌈

### CODE REVIEW

File: translations/generate.plugin.rulesaction.yml [hunk 1]
```diff
@@ -1,29 +0,0 @@
-description: 'Genera un plugin de acción de rules'
```
Comment (enzolutions) [inline translations/generate.plugin.rulesaction.yml:1]: @novia713 why this file was deleted?

Comment (novia713) [reply translations/generate.plugin.rulesaction.yml:1]: seems translate command deleted this and added `translations/generate.plugin.rules.action.yml`
note the dot between `rules` and `action`

i realy don't know which is the correct :-/

Comment (novia713) [reply translations/generate.plugin.rulesaction.yml:1]: seems in english the file is with dot also, so i guess this deleting and creating the new file with dot is correct ....

Comment (enzolutions) [reply translations/generate.plugin.rulesaction.yml:1]: @jmolivas do you have any comments here?

Comment (jmolivas) [reply translations/generate.plugin.rulesaction.yml:1]: We should use same as en language

Comment (jmolivas) [reply translations/generate.plugin.rulesaction.yml:1]: We can take a look at the command code to check the correct one

**Comment by enzolutions** [comment]
@novia713 we could close this ticket? or do you think we can solve the conflicts?

**Comment by novia713** [comment]
i think we can close this, because we already commited later translations, and each time it involves all the files.
i'm planning a next & more recent translations update, probably for the beginning of October.

**Comment by enzolutions** [comment]
Awesome @novia713, great contributions as always

**Pull request closed without merging**